### DISCUSSION TITLE
Why Laravel not adding make:view command?

### DISCUSSION THREAD

**Discussion by Nimaw** [laravel/framework#39994] [Ideas] [answered]
Hey guys, 
This is a big question for me as to why Laravel does not add this  command?

Example command usages: 
Create simple view:
`php artisan make:view ViewName`
Create view extending from another view:
`php artisan make:view childViewName --extends=ParentViewName`
Create view extending from another view with sections:
`php artisan make:view childViewNameWithSections --extends=ParentViewName --sections=title,content`

**Comment by Nimaw** [comment, marked as answer]
Hey guys,
I have solved problem my self:)
I've developed a package for that.
you can check my package [here](https://github.com/Nimaw/Lara-Commands).

**Reply by Sunhat** [reply]
Could you not make a PR for this instead?

**Reply by Nimaw** [reply]
Ummm,
That's good idea.
I'll do it, Thanks for you suggestion.

**Reply by dennisprudlo** [reply]
This was recently rejected in https://github.com/laravel/framework/pull/40131.

**Comment by shaedrich** [comment]
Your wishes have been heard: #48330
